private-templates-service/client/src/components/Common/SpinnerModal/styled.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { Spinner } from 'office-ui-fabric-react/lib/Spinner';
+import { COLORS } from '../../../globalStyles';
+
+export const CenteredSpinner = styled(Spinner)`
+  margin: auto;
+`;
+
+export const Dot = styled.div`
+  height: 30px;
+  width: 30px;
+  background-color: ${COLORS.WHITE};
+  border-radius: 50%;
```

Comment:
nit
 ask naomi how the spinner background should look, I think its more standard to have a square background.